src/screens/CategoriesFilterScreen/ListItem.js

```diff
@@ -0,0 +1,140 @@
+// @flow
+import React, { Component } from "react";
+import {
+  StyleSheet,
+  TouchableOpacity,
+  View,
+  Animated,
+  Dimensions,
+  Image
+} from "react-native";
+import Text from "../../components/Text";
+import { whiteColor, blackColor } from "../../constants/colors";
+import whiteCheck from "../../../assets/images/whiteCheck.png";
+import blackCheck from "../../../assets/images/blackCheck.png";
+import type { EventCategory } from "../../data/event";
+
+type ListItemProps = {
+  category: EventCategory,
+  onPress: Function,
+  selected: boolean
+};
+
+class ListItem extends Component<ListItemProps, { textWidth: number }> {
+  static defaultProps = {
+    onPress: () => {}
+  };
+
+  constructor(props: ListItemProps) {
+    super(props);
+
+    // No initial animation
```

Comment:
This comment confuses me slightly. No animation ... lets start an animation 😄 
It's because of the `duration` set to 0, right?

Reply:
Right, but it was trying to point out the reason why it's set to 0. Long story short, the `selected` state is coming through props so it would animate all selected categories when you reopen the panel. There's no need for that hence the code & comment.